src/WebJobs.Extensions.DurableTask/DurableOrchestrationClient.cs

```diff
@@ -90,8 +90,9 @@ public override async Task<string> StartNewAsync(string orchestratorFunctionName
                 throw new ArgumentException($"Instance ID lengths must not exceed {MaxInstanceIdLength} characters.");
             }
 
+            var dedupeStatuses = this.GetStatusesNotToOverride();
```

Comment:
`dedupestatuses` is the parameter name for `CreateOrchestrationInstanceAsync()`, but it is not a clear representation of what the variable means in this context. Maybe `nonOverwriteableStatuses`